html-dialog/html-dialog-demos/src_hd/ch/ivyteam/htmldialog/demo/output/DynamicDialogDemo/DynamicDialogDemo.xhtml

```diff
@@ -0,0 +1,31 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+    xmlns:f="http://xmlns.jcp.org/jsf/core"
+    xmlns:h="http://xmlns.jcp.org/jsf/html"
+    xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+    xmlns:ic="http://ivyteam.ch/jsf/component"
+    xmlns:p="http://primefaces.org/ui"
+    xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+ <ui:composition template="/layouts/HtmlDemo.xhtml">
+    <ui:param name="centerHeader" value="Dynamic Dialog Demo" />
+    <ui:define name="breadcrumb">Dynamic Dialog Demo</ui:define>
+    <ui:define name="content">
+    
+    <h:panelGroup layout="block">
+        <p>Dynamic dialogs allows you to open xhtml files in new dialogs.
```

Comment:
please use a spell-checker to correct the grammer.
e.g. https://app.grammarly.com/docs/new